src/native/corehost/setup.cmake

```diff
@@ -50,3 +50,9 @@ else()
         add_definitions(-DREPO_COMMIT_HASH="${CLI_CMAKE_COMMIT_HASH}")
     endif()
 endif()
+
+if("${CLI_CMAKE_FALLBACK_RID}" STREQUAL "")
```

Comment:
nit: Can we name it something like `CLI_CMAKE_FALLBACK_RID_OS` and `FALLBACK_HOST_RID_OS` instead? I know the existing define just had RID, but I think it would be helpful to have some indication that it needs to only be the OS portion, not any full RID.